app/plugin_api.go

```diff
@@ -190,12 +190,12 @@ func (api *PluginAPI) GetChannel(channelId string) (*model.Channel, *model.AppEr
 	return api.app.GetChannel(channelId)
 }
 
-func (api *PluginAPI) GetChannelByName(teamId, name string) (*model.Channel, *model.AppError) {
-	return api.app.GetChannelByName(name, teamId)
+func (api *PluginAPI) GetChannelByName(teamId, name string, includeDeleted bool) (*model.Channel, *model.AppError) {
```

Comment:
is this a binary incompatible change?

Reply:
Yes it is, but was the decision made in the developers meeting. We (joram and I) did a proposal to use other approach (based on Option parameter pattern) but after the discuss it, the final decision was to use an extra simple parameter.